src/HTMLConverter.php

```diff
@@ -58,19 +58,42 @@ public function __construct()
     /**
      * Convert an Record collection into a DOMDocument.
      *
-     * @param array|Traversable $records the tabular data collection
+     * @param array|Traversable $records      The tabular data collection
+     * @param array             $headerRecord An optional array of headers to output to the table using `<thead>` and `<th>` elements
+     * @param array             $footerRecord An optional array of footers to output to the table using `<tfoot>` and `<th>` elements
+     *
+     * @return string
      */
-    public function convert($records): string
+    public function convert($records, array $headerRecord = [], array $footerRecord = []): string
     {
-        /** @var DOMDocument $doc */
-        $doc = $this->xml_converter->convert($records);
+        // If we do not want to add any headers or records, then we can default entirely to the XML converter
```

Comment:
This comment is unnecessary the code is self explaining in itself

Reply:
removed